File: PYTHON/barang.py
class Barang:
    def __init__(self, id: int, nama: str, kategori: str, harga: float, stok: int):
        self._id = id
        self._nama = nama
        self._kategori = kategori
        self._harga = harga
        self._stok = stok

    def getId(self): return self._id
    def getNama(self): return self._nama
    def getKategori(self): return self._kategori
    def getHarga(self): return self._harga
    def getStok(self): return self._stok

    def setNama(self, nama: str): self._nama = nama
    def setKategori(self, kategori: str): self._kategori = kategori
    def setHarga(self, harga: float): self._harga = harga
    def setStok(self, stok: int): self._stok = stok

File: PHP/Barang.php
<?php

class Barang {
    private int $id;
    private string $nama;
    private string $kategori;
    private float $harga;
    private string $gambar;

    public function __construct(int $id, string $nama, string $kategori, float $harga, string $gambar) {
        $this->id = $id;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->gambar = $gambar;
    }

    // Getter
    public function getId(): int { return $this->id; }
    public function getNama(): string { return $this->nama; }
    public function getKategori(): string { return $this->kategori; }
    public function getHarga(): float { return $this->harga; }
    public function getGambar(): string { return $this->gambar; }

    // Setter
    public function setNama(string $nama): void { $this->nama = $nama; }
    public function setKategori(string $kategori): void { $this->kategori = $kategori; }
    public function setHarga(float $harga): void { $this->harga = $harga; }
    public function setGambar(string $gambar): void { $this->gambar = $gambar; }
}
